contrib/pyln-client/pyln/client/gossmap.py

```diff
@@ -53,6 +53,7 @@ def __init__(self, channel: 'GossmapChannel', direction: int,
         self.htlc_maximum_msat: Optional[int] = fields.get('htlc_maximum_msat', None)
         self.fee_base_msat: int = fields['fee_base_msat']
         self.fee_proportional_millionths: int = fields['fee_proportional_millionths']
+        self.disabled = fields['channel_flags'] & 2 > 0
```

Comment:
The more things we cache, the more likely it is we get some inconsistency at some point. In this case we should likely rather refer to the latest update, and make this a `@property` otherwise we will likely miss an update to this cached value at some point. Then again, since we use `fields` rather than referring to the underlying update it is likely ok here.

Reply:
We read through gossip from start to end and overwrite with each message (if its not the first occurence of a node or channel).